.eslintignore

```diff
@@ -0,0 +1 @@
+**/__tests__/**
```

Comment:
Why would you ignore test files?

Reply:
I wanted quickly to solve issues with jest and eslint(`describe` and `it`, `expect` undefined)... but better to add eslint plugin for jest... will do